test/nodes/decorators/Not.test.ts

```diff
@@ -12,8 +12,8 @@ const Action = Behavior.Action;
 const Not = Behavior.decorators.Not;
 
 class EchoAction extends Action<DroneState, string> {
-	onEvent(state: DroneState, event: string) {
-		return event;
+	async onEvent(state: DroneState, event: string) {
+		return event as any;
```

Comment:
Not sure on the intention here. The event type is "string" but we then also return it -- events and result codes being the same seems odd to me.